### PR TITLE
Move linked issues adjacent to their linked PRs

### LINKED ISSUE #13: Sprint automation: when a PR links to an issue, move the issue across the sprint board with the PR
The Enarx sprint framework treats pull requests differently from issues: they have their own workflow for being reviewed and merged.

Pull requests frequently link to issues in order to automatically close them. However, this means that when a PR is added to the sprint board in order to track its progress, the task encapsulated by its linked issue is duplicated (both the issue and the linked PR are present, when they refer to the same task).

To minimize this duplication, we should have issues move alongside PRs when they are linked.

### PR DESCRIPTION
**Pull request by mbestavros**
Adds an action to move issues next to their linked PRs on the sprint board.

Resolves #13.

### CODE REVIEW

File: enarx-adjacent-linked-issues [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# Cache all project cards in a dictionary for easy lookup (and to avoid
+# redundant network requests).
+def cache_board(project):
+    card_cache = {}
+    for column in project.get_columns():
+            for card in column.get_cards():
+                content = card.get_content()
+                if content:
+                    card_cache[content.id] = (card, column)
+    return card_cache
```
Comment (npmccallum) [inline enarx-adjacent-linked-issues:31]: Share this code?

**Comment by mbestavros** [comment]
My latest push adds a shared `common.py` with functions used across actions. Also replicated in https://github.com/enarx/bot/pull/10.

**Comment by mbestavros** [comment]
My latest push fixes a duplicate bug with `get_related_issues`.

**Comment by mbestavros** [comment]
We've since moved on from REST API v3, so this will need to be rewritten. Closing in favor of a new PR eventually.

**Pull request closed without merging**